### PR TITLE
Folspringb 61

### PR DESCRIPTION
**Pull request by arin-epm**

### CODE REVIEW

**Review by Arghya22 — APPROVED** [review]

**Review by gurleenkaurbp — APPROVED** [review]

File: mod-spring-petstore/src/main/java/org/folio/petstore/domain/service/impl/PetServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  public void createPet(PetDTO petDTO) {
+    Pet pet = mapper.toPetEntity(petDTO);
+    petRepository.save(pet);
+  }
+
+  @Override
+  public List<PetDTO> listPetDTOs(Integer limit) {
+    Pageable pageable = Pageable.ofSize(limit);
+    List<PetDTO> petDTOList = mapper.toPetDTOList(petRepository.findAll(pageable));
+    return null;
```
[the remark this answers is not in the document]
Comment (arin-epm) [reply mod-spring-petstore/src/main/java/org/folio/petstore/domain/service/impl/PetServiceImpl.java]: resolved

File: mod-spring-petstore/src/main/java/org/folio/petstore/controller/PetsController.java [hunk 2]
```diff
@@ -1,59 +1,45 @@
...
 
 import lombok.RequiredArgsConstructor;
-import org.folio.petstore.domain.dto.Pet;
-import org.folio.petstore.rest.resource.PetsApi;
+import org.folio.petstore.domain.dto.PetDTO;
+import org.folio.petstore.domain.service.impl.PetServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (TarasSpashchenko) [inline mod-spring-petstore/src/main/java/org/folio/petstore/controller/PetsController.java:8]: Please do not use wildcards

**Review by TarasSpashchenko — CHANGES_REQUESTED** [review]
Please use API first approach.
It means that OpenAPI specification should be created and DTOs and Controllers will be generated for you.

File: mod-spring-petstore/src/main/java/org/folio/petstore/controller/PetsController.java (hunk 2 above)
Comment (arin-epm) [reply mod-spring-petstore/src/main/java/org/folio/petstore/controller/PetsController.java]: Changes added for OpenAPI specification. Requesting your review.

**Pull request merged**